zookeeper-server/src/main/java/org/apache/zookeeper/server/ZooKeeperServer.java

```diff
@@ -231,6 +231,14 @@ public ZooKeeperServer(FileTxnSnapLog txnLogFactory, int tickTime)
         this(txnLogFactory, tickTime, -1, -1, -1, new ZKDatabase(txnLogFactory));
     }
 
+    public Deque<ChangeRecord> getOutstandingChanges() {
+        return outstandingChanges;
```

Comment:
Can we make this field private as we are encapsulating it?

Reply:
I've changed the unit test so it doesn't need the getter anymore. So this field stays as is. Alos reverted the code to access the field directly (which is not nice) instead of calling the getter